src/sql/src/error.rs

```diff
@@ -370,6 +370,38 @@ pub enum Error {
         #[snafu(implicit)]
         location: Location,
     },
+
+    #[cfg(feature = "enterprise")]
+    #[snafu(display("Missing `{}` clause", name))]
+    MissingClause {
+        name: String,
+        #[snafu(implicit)]
+        location: Location,
+    },
+
+    #[cfg(feature = "enterprise")]
+    #[snafu(display("Unrecognized trigger webhook option key: {}", key))]
+    InvalidTriggerWebhookOption {
+        key: String,
+        #[snafu(implicit)]
+        location: Location,
+    },
+
+    #[cfg(feature = "enterprise")]
+    #[snafu(display("Failed to convert int"))]
```

Comment:
We will need more context in the error message.

Reply:
Done in https://github.com/GreptimeTeam/greptimedb/pull/6197/commits/db89505c7156b73f688b92982a3a05a99d2c188d

Reply:
Refine in https://github.com/GreptimeTeam/greptimedb/pull/6197/commits/4582c47e0505f58a8efb4523c93f5e5a90d67e96